modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -248,6 +248,11 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
     float power;
 #endif
 
+#ifdef HAVE_TENGINE
+    graph_t graph;
+    bool tengine_ret;
```

Comment:
do we need `tengine_ret` here? If it's just return code, we can declare it locally inside functions when needed

Reply:
i try to remove it .